File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/auth/TokenUserPassScmAuth.java
package com.tencent.devops.scm.api.pojo.auth;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TokenUserPassScmAuth implements IScmAuth {

    public static final String CLASS_TYPE = "TOKEN_USER_PASS";

    private String token;
    private String username;
    private String password;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/auth/TokenUserPassScmAuth.kt
package com.tencent.devops.scm.api.pojo.auth

class TokenUserPassScmAuth(
    val token: String,
    val username: String,
    val password: String
) : IScmAuth {
    companion object {
        const val CLASS_TYPE = "TOKEN_USER_PASS"
    }
}
